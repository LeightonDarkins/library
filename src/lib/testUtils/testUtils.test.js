import { classListAsArray } from './index'

const mockElement = {
  classList: {
    '0': 'test-class-1',
    '1': 'test-class-2',
    '1test': 'another-class'
  }
}

describe('testUtils', () => {
  describe('classListAsArray', () => {
    it('returns the correct array for a given element', () => {
      const result = classListAsArray(mockElement)

      expect(result).toMatchSnapshot()
    })
  })
})