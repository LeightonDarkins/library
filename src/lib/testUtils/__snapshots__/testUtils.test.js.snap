// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`testUtils classListAsArray returns the correct array for a given element 1`] = `
Array [
  "test-class-1",
  "test-class-2",
  "another-class",
]
`;
